extensions/sql-database-projects/src/models/project.ts

```diff
@@ -225,6 +227,13 @@ export class Project implements ISqlProject {
 			}
 		}
 
+		if (this.isSdkStyleProject) {
+			// remove any pre/post deploy scripts that were specified in the sqlproj so they aren't counted twice
```

Comment:
you are also removing the none deploy scripts but not mentioned in the comments, which one is correct?

Reply:
updated the comment to also mention none deploy scripts